src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,11 +32,19 @@
 public class DataServlet extends HttpServlet {
   private final List<Object> commentsList = new ArrayList<>();
   private Key currentTermKey;
-  private LanguageServiceClient languageService;
   private final DatastoreService db = DatastoreServiceFactory.getDatastoreService();
+  private LanguageServiceClient languageService;
 
   // Will re-add constructor later for testing.
 
+  public DataServlet() throws IOException {
```

Comment:
This might cause an error like before. I think my code is blocking you, try commenting out all my graph related code and depoly to see if it works.

Reply:
Yeah I was thinking about that rip, will comment our your stuff and see if it still works correctly.

Reply:
It works!!! :)